src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx

```diff
@@ -7,21 +7,72 @@ import * as utils from '@/utils'
 import Link from 'next/link'
 
 export const AlreadyPaidLinkView = ({ requestLinkData }: { requestLinkData: _consts.IRequestLinkData | undefined }) => {
+
+    const chainName = consts.supportedPeanutChains && consts.supportedPeanutChains.find((chain) => chain.chainId == requestLinkData?.chainId)?.name
+    const tokenSymbolAvailable: boolean = requestLinkData?.tokenSymbol != undefined
+    const tokenAmountAvailable: boolean = requestLinkData?.tokenAmount != undefined
+    const chainAvailable: boolean = requestLinkData?.chainId != undefined
+    const recipientAddressAvailable: boolean = requestLinkData?.recipientAddress != undefined
+    const dataAvailable: boolean = tokenSymbolAvailable || tokenAmountAvailable || chainAvailable || recipientAddressAvailable
+    
     return (
         <div className="flex w-full flex-col items-center justify-center gap-6 py-2 pb-20 text-center">
             <label className="text-h2">Sorry, this link has already been paid.</label>
-            <label className="text-h8 font-bold ">
-                This link previously requested{' '}
-                {requestLinkData?.tokenSymbol ??
```

Comment:
thought: there is loss of functionality here

Before we were trying to get a token symbol if there is none from the data. Now we just trust the data we get. I actually like the new way better but it warrants fixing why sometimes we don't have data from the request link itself. In the short term this change WILL result in some data previously shown to not be shown anymore